variants/uPesy_edu_esp32/pins_arduino.h

```diff
@@ -0,0 +1,62 @@
+#ifndef Pins_Arduino_h
+#define Pins_Arduino_h
+
+
+#include <stdint.h>
+
+#define EXTERNAL_NUM_INTERRUPTS 16
+#define NUM_DIGITAL_PINS        40
+#define NUM_ANALOG_INPUTS       16
+
+#define analogInputToDigitalPin(p)  (((p)<20)?(analogChannelToDigitalPin(p)):-1)
+#define digitalPinToInterrupt(p)    (((p)<40)?(p):-1)
+#define digitalPinHasPWM(p)         (p < 34)
```

Comment:
the above defines (lines 7-13) are no longer required and will be automatically provided by the base platform.